Gemfile.lock

```diff
@@ -245,33 +247,34 @@ GEM
     method_source (1.0.0)
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2019.1009)
-    mimemagic (0.3.4)
+    mime-types-data (3.2020.0512)
+    mimemagic (0.3.5)
     mini_magick (4.10.1)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
     minitest (5.14.1)
     msgpack (1.3.3)
     mysql2 (0.5.3)
-    net-scp (2.0.0)
-      net-ssh (>= 2.6.5, < 6.0.0)
-    net-ssh (5.2.0)
+    net-scp (3.0.0)
```

Comment:
### [net-scp](https://github.com/net-ssh/net-scp)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 2.0.0 → 3.0.0

[_[change-log](https://github.com/net-ssh/net-scp/blob/master/CHANGES.txt), ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **16** commits. See the full changes on [the compare page](https://github.com/net-ssh/net-scp/compare/v2.0.0...v3.0.0).

These are the first 10 commits:
- (f7a308d) [Set encoding to 'BINARY' when creating a StringIO - this wil…](https://github.com/net-ssh/net-scp/commit/f7a308d7ea0de4364564527b444970b40cecb5ae)
- (85c732c) [adding recursive flag](https://github.com/net-ssh/net-scp/commit/85c732cadaf75473d86e3b092770463d60b1f355)
- (3c41ea6) [Update README.doc, Typo](https://github.com/net-ssh/net-scp/commit/3c41ea632799889fd1b26b395bd6fc18fd0f8d63)
- (6cd391f) [Fix a typo in README](https://github.com/net-ssh/net-scp/commit/6cd391faa2d82dfa075675ead407028a73f6a97f)
- (4b4d9c4) [Add changelog_uri to metadata to easily link from rubygems.o…](https://github.com/net-ssh/net-scp/commit/4b4d9c46cd155d5acafab7c9966f4cb11c43e5c7)
- (066175f) [Remove deprecated option from gemspec](https://github.com/net-ssh/net-scp/commit/066175fb20959bcd830689eb228990655102137b)
- (e5e9ffb) [Merge pull request #48 from mjgiarlo/patch-1](https://github.com/net-ssh/net-scp/commit/e5e9ffb6b2a935d60d69769db28ff74ffd6d613c)
- (e6a797c) [Merge pull request #45 from nickhammond/patch-1](https://github.com/net-ssh/net-scp/commit/e6a797c4bd7dd3525847cb1d52ebda7e7c0d51c3)
- (1564c84) [Merge pull request #44 from Inspyre-Technologies/master](https://github.com/net-ssh/net-scp/commit/1564c84f40c9fab970510c0c4a32a54b68aabbeb)
- (89b7dac) [Merge pull request #22 from vinyar/master](https://github.com/net-ssh/net-scp/commit/89b7dac3d242ec9e9120353de20811e8845dca51)


</details>